docs/guides/basics/create_new.md

```diff
@@ -4,19 +4,31 @@ repositories.
 
 To create a new Project, visit [www.floydhub.com/projects](https://www.floydhub.com/projects) and click on the "New Project" button on the top right hand corner.
 
-![Create new project](../../img/create_new_project.jpg)
+![Create new project](../../img/create_project/create_project.png)
 
-Give the project a name and an apt description.
+Give the project a name and a description.
+
+The `Owner` field defines the user space under which this project belong. If you are inside a Team select the name of your Team to create the project under the Team space.
```

Comment:
Simplified to:
The `Owner` field defines the user namespace to which this project belongs. If you are part of a Team select the name of your Team.